lighthouse-core/scripts/i18n/collect-strings.js

```diff
@@ -13,7 +13,7 @@ const path = require('path');
 const esprima = require('esprima');
 
 const LH_ROOT = path.join(__dirname, '../../../');
-const UISTRINGS_REGEX = /UIStrings = (.|\s)*?\};\n/gim;
```

Comment:
If this regex is global it tries to match any content following `module.exports.UIStrings = UIStrings;` and then backtracks. If there's some content after that line the collect-strings script hangs.

Reply:
btw this no longer collects all matches, but that's fine cuz we only ever use the first instance.